packs/data/spell-effects.db/spell-effect-unblinking-flame-emblem.json

```diff
@@ -0,0 +1,39 @@
+{
+    "_id": "PkEcH06RRgiStIa4",
+    "img": "systems/pf2e/icons/spells/flame-wisp.webp",
+    "name": "Spell Effect: Unblinking Flame Emblem",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by <em>@UUID[Compendium.pf2e.spells-srd.Unblinking Flame Emblem]{Unblinking Flame Emblem}</em></p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 3
+        },
+        "level": {
+            "value": 7
+        },
+        "rules": [],
```

Comment:
Why have an effect if it isn't carrying the rule info?

Reply:
To mark the target, and as a placeholder to contain the rule info once it's supported. I can remove it if you prefer.

Reply:
Removed the effect for now.